### PR TITLE
build images using Buildkit

### PR DESCRIPTION
**Pull request by ryanpetrello**
https://docs.docker.com/develop/develop-images/build_enhancements/

@shanemcd how does this make you feel?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -596,7 +597,7 @@ docker-compose-clean: awx/projects
 # Base development image build
 docker-compose-build:
 	ansible-playbook installer/dockerfile.yml -e build_dev=True
-	docker build -t ansible/awx_devel \
+	$(COMPOSE_BUILDENV) docker build -t ansible/awx_devel \
```
Comment (shanemcd) [inline Makefile]:
```suggestion
	DOCKER_BUILDKIT=1 docker build -t ansible/awx_devel \
```

File: Makefile [hunk 2]
```diff
@@ -18,6 +18,7 @@ PYCURL_SSL_LIBRARY ?= openssl
 # NOTE: This defaults the container image version to the branch that's active
 COMPOSE_TAG ?= $(GIT_BRANCH)
 COMPOSE_HOST ?= $(shell hostname)
+COMPOSE_BUILDENV ?= $(shell [[ "$OSTYPE" == "darwin"* ]] && echo "" || echo "DOCKER_BUILDKIT=1")
```
Comment (shanemcd) [inline Makefile]:
```suggestion

```

**Review by shanemcd — CHANGES_REQUESTED** [review]
I tested w/ Docker Desktop and BuildKit works fine there.

**Review by shanemcd — APPROVED** [review]

**Comment by ryanpetrello** [comment]
regate

**Pull request merged**